packages/components/src/components/video/video.stories.ts

```diff
@@ -13,13 +13,18 @@ const videoConstant: ConstantDefinition = {
   type: 'slot',
   name: 'default',
   value:
-    '<video controls id="videoExample" style="min-width: 854px; min-height: 480px; width: 854px; height: 480px;"><source src="http://media.w3.org/2010/05/sintel/trailer.mp4" type="video/mp4" />Your browser does not support the video tag.</video>'
+    '<video controls id="video-example" class="w-full aspect-video"><source src="http://media.w3.org/2010/05/sintel/trailer.mp4" type="video/mp4" />Your browser does not support the video tag.</video>'
+};
+const imageConstant: ConstantDefinition = {
+  type: 'slot',
+  name: 'default',
+  value: '<img class="w-[400px] aspect-video object-cover" src="./placeholders/generic.jpg" />'
 };
 const posterConstant: ConstantDefinition = {
   type: 'slot',
   name: 'poster',
   value:
-    '<img slot="poster" alt="poster" src="https://www.blender.org/wp-content/uploads/2020/10/robin-tran-redautumnforest_pr1.jpg" />'
+    '<img slot="poster" alt="poster" class="w-WHATEVER-WIDTH aspect-video cover" src="./placeholders/architecture.jpg" />'
```

Comment:
@mariohamann was this `w-WHATEVER-WIDTH` really what you expected? If yes, why? Would love to then see a description here.